index.html

```diff
@@ -4,9 +4,152 @@
+        </div>
+    </nav>
+    <!-- sidebar -->
+    <div class="sidebar">
+        <div class="shortcut-links">
+            <a href=""><img src="images/home.png" alt="home-icon"><p>Home</p></a>
+            <a href=""><img src="images/explore.png" alt="explore-icon"><p>Explore</p></a>
+            <a href=""><img src="images/subscriprion.png" alt="subscriprion-icon"><p>Subscription</p></a>
+            <a href=""><img src="images/library.png" alt="library-icon"><p>Library</p></a>
+            <a href=""><img src="images/history.png" alt="history-icon"><p>History</p></a>
+            <a href=""><img src="images/playlist.png" alt="playlist-icon"><p>Playlist</p></a>
+            <a href=""><img src="images/messages.png" alt="messages-icon"><p>Messages</p></a>
+            <a href=""><img src="images/show-more.png" alt="show-more-icon"><p>Show-more</p></a>
+            <hr>
+        </div>
+        <div class="subscribed-list">
+            <h3>SUBSCRIBED</h3>
+            <a href=""><img src="images/Jack.png"><p>Jack Nicholson</p></a>
+            <a href=""><img src="images/simon.png"><p>Simon Baker</p></a>
+            <a href=""><img src="images/tom.png"><p>Tom Hardy</p></a>
+            <a href=""><img src="images/megan.png" ><p>Megan Ryan</p></a>
+            <a href=""><img src="images/cameron.png"><p>Cameron Diaz</p></a>
+        </div>
+    </div>
+    <!---Main-->
+    <div class="container">
+        <div class="banner">
+            <img src="images/banner.png">
+        </div>
+        <div class="list-container">
+            <div class="vid-list">
+                <a href=""><img src="images/thumbnail1.png" class="thumbnail"></a>                
+                <div class = "flex-div">
+                    <img src="images/Jack.png">
+                    <div class="vid-info">
+                        <a href="">Best channel to learn coding</a>
+                        <p>Adorer Tutorials</p>
+                        <p>15k Views &bull; 5 days</p>
+                    </div>
+                </div>
+            </div>
+            <div class="vid-list">
+                <a href=""><img src="images/thumbnail2.png" class="thumbnail"></a>                
+                <div class = "flex-div">
+                    <img src="images/Jack.png">
+                    <div class="vid-info">
+                        <a href="">Best channel to learn coding</a>
+                        <p>Adorer Tutorials</p>
+                        <p>15k Views &bull; 5 days</p>
+                    </div>
+                </div>
+            </div>
+            <div class="vid-list">
+                <a href=""><img src="images/thumbnail3.png" class="thumbnail"></a>                
+                <div class = "flex-div">
+                    <img src="images/Jack.png">
+                    <div class="vid-info">
+                        <a href="">Best channel to learn coding</a>
```

Comment:
Any of the Hx tags will be nice here. You might consider using (h3-h6)

Reply:
- [ ] Please always keep in mind while doing code review. kindly communicate politely. suggest instead of giving order this way the student feels comfortable and doesn't not frustrate. and also try to give WHY you requested changes adding more details you let the student understand better that what problem is about.
### kindly check this👇
![review](https://user-images.githubusercontent.com/66992901/200529861-2ae37d3b-9350-4234-8e45-d1c451fd05f7.PNG)